openstates/scrape/base.py

```diff
@@ -182,10 +190,11 @@ def save_object(self, obj):
 
             if self.realtime:
 
+                self.output_file_path = str(file_path_)
+
                 s3 = S3FileSystem(anon=False)
 
-                S3_FULL_PATH = settings.S3_REALTIME_BASE + str(file_path_)
-                self.s3_output_path = S3_FULL_PATH
+                S3_FULL_PATH = settings.S3_REALTIME_BASE + self.output_file_path
```

Comment:
This string isn't created with an f-string. Just a nit, but perhaps we fix that?

Reply:
Yeah, would fix this thanks. Still in draft,  will update